app/views/pages/_editor_guide_text.html.erb

```diff
@@ -13,13 +13,13 @@
     </ul>
 
     <h2><u><strong>Front Matter</strong></u></h2>
-    <p>Custom variables set for each post, located between the triple-dashed lines in your editor. Here are a list of possibilities:</p>
+    <p>Custom variables set for each post, located between the triple-dashed lines in your editor. Here is a list of possibilities:</p>
     <ul>
-      <li><strong>title:</strong> title of your article</li>
+      <li><strong>title:</strong> the title of your article</li>
       <li><strong>published:</strong> boolean that determines whether or not your article is published</li>
       <li><strong>description:</strong> description area in Twitter cards and open graph cards</li>
-      <li><strong>tags:</strong> max of four tags, needs to be comma-separated</li>
-      <li><strong>canonical_url:</strong> link for canonical version of content</li>
+      <li><strong>tags:</strong> max of four tags, needs to be comma-separated or space-separated</li>
```

Comment:
Tags can't be space-separated. I've verified.

Reply:
Space separated works for me, see GIF...
![devto-space-separated](https://user-images.githubusercontent.com/1869717/46425643-1108af80-c6f1-11e8-8984-93bf33a3c2cb.gif)

Reply:
Ah that's why. 
![](https://cl.ly/d71e982b4804/Screen%20Recording%202018-10-03%20at%2012.52%20PM.gif)

Since we can't expect everybody to be adding a hashtag to their tags, making comma-separated mandatory is a safer choice.

Reply:
Agreed. Updated this PR to remove space separated from editor guide.